lib/classes/PluginManager.js

```diff
@@ -75,20 +76,15 @@ class PluginManager {
   loadServicePlugins(servicePlugs) {
     const servicePlugins = Array.isArray(servicePlugs) ? servicePlugs : [];
 
+    // eslint-disable-next-line no-underscore-dangle
+    module.paths = Module._nodeModulePaths(process.cwd());
+
     // we want to load plugins installed locally in the service
     if (this.serverless && this.serverless.config && this.serverless.config.servicePath) {
-      module.paths.unshift(
-        path.join(this.serverless.config.servicePath, 'node_modules'),
-        path.join(this.serverless.config.servicePath, '.serverless_plugins')
-        );
+      module.paths.unshift(path.join(this.serverless.config.servicePath, '.serverless_plugins'));
     }
 
     this.loadPlugins(servicePlugins);
-
-    // restore module paths
-    if (this.serverless && this.serverless.config && this.serverless.config.servicePath) {
-      module.paths.shift();
```

Comment:
Yes, that's a good point. Thanks for pointing that out! I'll bring it back for now, but we might refactor this so that it's called in the constructor!